### PR TITLE
Feature/implement typescript

### PR DESCRIPTION
**Pull request by ankitpmi**
- Implemented typescript.
- Resolved type issues.
- Update readme file.
- Removed unused packages.

### CODE REVIEW

File: Tabbar/TabBar.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      );
+    }
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: Dimensions.get("window").width,
+  },
+  container2: { flex: 1, justifyContent: "center", alignItems: "center" },
```
Comment (Ruturaj2728) [inline Tabbar/TabBar.tsx:194]: Please give proper name to style

File: Tabbar/TabBar.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+          <Text>Please add tab data</Text>
+        </View>
+      );
+    }
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: Dimensions.get("window").width,
```
Comment (Ruturaj2728) [inline Tabbar/TabBar.tsx:192]: Please use constant for this

File: Tabbar/TabBar.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+                }}
+              >
+                <Path fill={tabBarBackgroundColor} {...{ d }} />
+              </AnimatedSvg>
+              <View style={StyleSheet.absoluteFill}>
+                <StaticTabbar {...this.props} {...{ tabs, value }} />
+              </View>
+            </View>
+            <SafeAreaView
+              style={[
```
Comment (Ruturaj2728) [inline Tabbar/TabBar.tsx:168]: If [] is not needed in style then remove it and update it accordingly

**Review by akashmi — APPROVED** [review]

File: Tabbar/TabBar.tsx [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+                ? tabBarContainerBackground
+                : "#fff",
+              position: "absolute",
+              bottom: containerBottomSpace ? containerBottomSpace : 0,
+              alignSelf: "center",
+              borderTopRightRadius,
+              borderTopLeftRadius,
+              borderBottomLeftRadius,
+              borderBottomRightRadius,
+              // height: height + 25
```
Comment (Ruturaj2728) [inline Tabbar/TabBar.tsx]: Remove commented code

**Review by Ruturaj2728 — APPROVED** [review]

**Pull request merged**